packages/flutter/lib/src/widgets/dismissible.dart

```diff
@@ -232,6 +238,12 @@ class Dismissible extends StatefulWidget {
   /// This defaults to [HitTestBehavior.opaque].
   final HitTestBehavior behavior;
 
+  /// Called when the dismiss threshold has been reached while dragging the widget.
+  /// When the dismiss thresold is reached, this function is called with the value
```

Comment:
```suggestion
  ///
  /// When the dismiss thresold is reached, this function is called with the value
```

Reply:
Thank you, I have done the suggested changes, that definitely make it better. Hopefully they are okay!